spec/factories/ext_management_system.rb

```diff
@@ -234,6 +234,18 @@
     provider_region "us-east-1"
   end
 
+  factory :ems_amazon_storage_manager_ebs,
```

Comment:
I would try _not_ to add more provider specific factories. See below for another solution to test this